addons/website/models/res_config_settings.py

```diff
@@ -120,11 +128,18 @@ def inverse_has_default_share_image(self):
         if not self.has_default_share_image:
             self.social_default_image = False
 
+    def inverse_has_recaptchav3(self):
+        if not self.has_recaptchav3:
+            self.recaptcha_public_key = False
+            self.recaptcha_private_key = False
+            self.recaptcha_min_score = self.env['website']._fields['recaptcha_min_score'].default('website')
```

Comment:
use public and overridable method:
`self.website_id.default_get(['recaptcha_min_score'])['recaptcha_min_score']`